extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -715,6 +715,26 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
     return {'artc': artc.join() || null, 'ati': ati.join(), 'ard': ard.join()};
   }
 
+  /** @override */
+  getCustomRealTimeConfigMacros_() {
+    return {
+      PAGEVIEWID: () => {
```

Comment:
nit: () => Services.documentInfoForDoc(this.element).pageViewId,

Reply:
done